### ISSUE TITLE
mobile - locations - information populated below with no visual cue

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#61]
When the cities within the states are tapped, nothing on the page alerts the user that more information for that particular city/state has been populated below. On the browser; however, displaying of related info is no issue since the information is populated to the right of the cities/states. For mobile, I think something on the page should somehow illustrate to the user that the information has been populated below since it visually looks like nothing happens when a city is clicked/tapped.

<img width="342" alt="Screen Shot 2020-12-02 at 12 42 07 PM" src="https://user-images.githubusercontent.com/618307/100911752-9e94e500-349d-11eb-87d1-f6d9fa57a9b0.png">

Possible solutions

1. I'm thinking there should be a page alert to show the user that information has been populated below, text like "Alamosa, Colorado info below. Scroll down to view" could be displayed to the right of the link that was tapped. 
2. Another idea is displaying a down arrow link to the right of the selected city, and then when the user taps it, the page shifts down to the start of the additional info section.
3. Populating the information directly underneath the city that was selected. This could also be treated like the accordion/toggle view that is currently on the states. So the city itself can be closed after the information has been viewed by the user, just like they can currently close the state by selecting it again. I'm not sure if this would greatly affect the browser view though.**
4. Another option is to open the information per city in a new tab (but I think that could be a later change if necessary).

Having thought through these possible solutions, I personally prefer # 3 as it follows the same structure as the state interactive element that was recently implemented and prevents the user from having to scroll down the page to view the specific info related to that city. 

Let me know your thoughts or if the general consensus is that people using the mobile site will already know to scroll down to view further info.

**Comment by kimisgold** [comment]
@IdaAkiwumi I've restructured the mobile views so that a button toggles a right off canvas menu of filters. The menu only partially covers the main content area, so when the user selects an active filter, they should see the content populate underneath the menu. At the moment, the menu stays on screen until the user taps outside it or hits escape; my current instinct is to not auto-close the menu in case the user misclicks while browsing the filters. How does this feel?

**Comment by IdaAkiwumi** [comment]
This is a great treatment! It looks good to me and works well, however when you first land on the Locations page now, there is Lorem ipsum text in place of where the state information would be. Maybe this text could say something about how the information will be populated in that space, if text needs to be there.

<img width="362" alt="Screen Shot 2020-12-07 at 9 32 03 AM" src="https://user-images.githubusercontent.com/618307/101363362-53008380-386f-11eb-8c47-dd8ad138ad29.png">

**Comment by kimisgold** [comment]
Excellent, I'm glad this works. Thanks for the reminder about the text: we're waiting on the finalized content in #50 and #51.
